src/components/AppLayout/index.tsx

```diff
@@ -86,7 +86,8 @@ const overviewItems = [
 	getSiderMenuItem('Overview', '/', <OverviewIcon className='text-white' />),
 	getSiderMenuItem('Discussions', '/discussions', <DiscussionsIcon className='text-white' />),
 	getSiderMenuItem('Calendar', '/calendar', <CalendarIcon className='text-white' />),
-	getSiderMenuItem('News', '/news', <NewsIcon className='text-white' />)
+	getSiderMenuItem('News', '/news', <NewsIcon className='text-white' />),
+	getSiderMenuItem('Parachains', '/parachains', <NewsIcon className='text-white' />)
```

Comment:
does the parachain link use the same icon as the news link here ?

Reply:
Yeah I fixed it